jwst/model_blender/rules.py

```diff
@@ -0,0 +1,81 @@
+from datetime import time
+
+import numpy as np
+
+
+__all__ = ['RULE_FUNCTIONS', 'Blender', 'make_blender']
+
+
+def multi(vals):
+    """
+    This will either return the common value from a list of identical values
+    or 'MULTIPLE'
+    """
+    uniq_vals = list(set(vals))
+    num_vals = len(uniq_vals)
+    if num_vals == 0:
+        return None
+    if num_vals == 1:
+        return uniq_vals[0]
+    if num_vals > 1:
+        return "MULTIPLE"
+
+
+def _isotime(time_str):
+    hms = [float(i) for i in time_str.split(':')]
+    sec_ms = hms[2] - int(hms[2])
+    isotime = time(int(hms[0]), int(hms[1]), int(hms[2]), int(sec_ms * 1000000))
+    return isotime
+
+
+def bad_mintime(vals):
+    # this introduces floating point error for backwards compatibility
```

Comment:
I think it's fine to make this change in the same PR and just say it resolves both tickets, but I suppose that's a procedural question best left to Melanie and Tyler

Reply:
@melanieclarke @tapastro 

Would you like me to fix https://jira.stsci.edu/browse/JP-3732 in this PR? It would require updating regtests.

If we're updating regtests I'd also like to consider fixing the missing attribute creation mentioned in the PR description. It seems like a bug to me that blending models would create attributes in the input models (even if the created values match the defaults).

Reply:
Okay by me - better to fix the bug than go out of your way to replicate it!

Reply:
Although I do appreciate that you were able to get a clean set of regression tests first to make sure everything else was working.  Thanks for that!